2019-11-30 09:09:01.602 - [Info] MITM Version: 1.26
2019-11-30 09:09:01.609 - [Info] Auto Access Enabled: true
2019-11-30 09:09:01.609 - [Debug] [Init] Auto Access Barrier: true
2019-11-30 09:09:01.633 - [Info] SSH man-in-the-middle server for 172.20.0.6 listening on 0.0.0.0:10105
2019-11-30 09:10:09.977 - [Debug] [Connection] Attacker connected: 122.51.43.61 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-30 09:10:11.604 - [Debug] [Auth] Attacker 122.51.43.61 trying to authenticate with "password"
2019-11-30 09:10:11.609 - [Debug] [Auto Access] Attacker: 122.51.43.61, Threshold: 1, Attempts: 1
2019-11-30 09:10:11.611 - [Debug] [Auto Access] Compromising the honeypot
2019-11-30 09:10:11.611 - [Debug] [Auto Access] Adding the following credentials: "root:xxx147"
2019-11-30 09:10:12.046 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-30 09:10:12.048 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.6
2019-11-30 09:10:12.246 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-30 09:10:13.335 - [Debug] [Connection] Attacker connected: 122.51.43.61 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-30 09:10:17.218 - [Debug] [Auth] Attacker 122.51.43.61 trying to authenticate with "password"
2019-11-30 09:10:17.218 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-30 09:10:17.219 - [Debug] [Auth] Attacker: 122.51.43.61 has so far made 1 attempts. Remaining: 5 attempts
2019-11-30 09:10:17.789 - [Debug] [Connection] Attacker closed the connection
2019-11-30 09:10:18.280 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-30 09:23:30.198 - [Debug] [Connection] Client error on ssh server
2019-11-30 09:23:30.199 - [Debug] [Connection] Attacker closed connection
2019-11-30 09:23:30.206 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-30 09:23:30.206 - [Debug] [LXC] Container's OpenSSH server closed connection
